Description: Add PATH_PREFIX to redirectToMigratedURL
 Old grafana versions use this format, and it breaks with prometheus instances
 not mounted in the root directory.

--- a/web/ui/static/js/graph.js
+++ b/web/ui/static/js/graph.js
@@ -927,7 +927,7 @@
     });
   });
   var query = $.param(queryObject);
-  window.location = "/graph?" + query;
+  window.location = PATH_PREFIX + "/graph?" + query;
 }
 
 $(init);
